### DISCUSSION TITLE
Funds not arrived at Solana. seems get stuck.

### DISCUSSION THREAD

**Discussion by tcw1103** [wormhole-foundation/wormhole#217] [Problems / bugs] [answered]
Dear sir.

I transferred HBTC to Solana address from Eth address through https://www.wormholebridge.com/#/move?from=ETH&token=HBTC.

Transaction has completed on ETH side, 
Transaction Hash:0x7f8b931482aef286994078f77e5d1751955dfb6e4aed508ae93c633919b7ce7e

However, on solana side,there is no hbtc minted for me.

my eth wallet:0xf53fEaeB035361c046e5669745695E450ebb4028
my sollet wallet:EcNJhDx7G7Rq6nwXdwoeViPym9DHgEVL1eJ6cgoxeMee

i found the following transaction,seems related to my case. 

https://explorer.solana.com/tx/4kidi3FmTEeaGYs7TcDrVYKWCs5coTr8zMZ5ii2NQcEXa7WXi2CyCrYR47ywmfgFkpMkJBCjiePXKtUB6Q75pvzM

Please help to resolve it . Thank you!

**Comment by hendrikhofstadt** [comment, marked as answer]
Completed in https://explorer.solana.com/tx/5g2RoNFPbj1j3PFrsyXCJCgDbL3i85eRMmjSrSCTRjWJFdKz3D9EH8PTvmF2qXfnQ87zCPz7Q8qP5SSWPmTu2SJ8
